app/boards/shields/murphpad/murphpad.conf

```diff
@@ -0,0 +1,9 @@
+# Uncomment to turn on logging, and set ZMK logging to debug output
+# CONFIG_ZMK_USB_LOGGING=y
+
+# Uncomment both to enable encoder
+CONFIG_EC11=y
+CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+
+# Uncomment to enable OLED
+CONFIG_ZMK_DISPLAY=y
```

Comment:
Just want to confirm that every murphpad has a display and encoder and isn't an optional add-on? If they're add-ons they should be commented out by default, but it seems like they're included from what I can see?

Reply:
They are included in every kit. The encoder IS optionally a switch instead, but the default for people is including it (and the guide instructs as much). I left them as options here in case they don't want to use the included encoder OR they want to remove the OLED code to save some battery life or whatever.